src/main/resources/db/changelog/alert/4.0.0/changelog-configuration-stored-procedures.xml

```diff
@@ -364,4 +364,26 @@
             $$;
         </createProcedure>
     </changeSet>
-</databaseChangeLog>
\ No newline at end of file
+    <!-- Create job migration procedure -->
+    <changeSet author="psantos" id="1545000000000-4">
+        <createProcedure
+                dbms="h2"
+                relativeToChangelogFile="true"
+                schemaName="ALERT"
+                procedureName="INSERT_FIELD_VALUE"
+        >
+            CREATE ALIAS INSERT_GLOBAL_FIELD_VALUE AS $$
+                java.lang.Void insertGlobalFieldValue(final java.sql.Connection connection, final java.lang.String descriptor_name, final java.lang.String fieldKey, final java.lang.String fieldValue) throws java.sql.SQLException {
+                    if (fieldValue != null &amp;&amp; !fieldValue.isBlank()) {
+                        try (final java.sql.Statement insertIntoFieldValues = connection.createStatement()) {
+                            final java.lang.String insertStatement = java.lang.String.format("INSERT INTO ALERT.FIELD_VALUES (CONFIG_ID, FIELD_ID, FIELD_VALUE) VALUES (GET_ID_FOR_REGISTERED_DESCRIPTOR_NAME('%s'), GET_FIELD_ID_FOR_SOURCE_KEY('%s'), '%s')",
```

Comment:
I think `GET_ID_FOR_REGISTERED_DESCRIPTOR_NAME(...)` returns the ID from the REGISTERED_DESCRIPTORS table, so this won't be the CONFIG_ID.